changelog.d/20250512_144528_max.tuecke_sc_26346_remove_base_path_from_clients.rst

```diff
@@ -0,0 +1,5 @@
+Removed
+~~~~~~~
+
+- SDK clients no longer define a ``base_path`` attribute which they prefix to paths.
```

Comment:
I would rephrase this to make it clear that the `base_path` isn't just "not defined", but in fact the prefixing logic has been completely removed.

Reply:
I think that was the phrasing I suggested -- so I want to be explicit that I'm happy for us to rephrase if this seems unclear.
Would you rather see two bullet points, rather than two sentences?

```rst
Removed
~~~~~~~

- SDK clients no longer define a ``base_path`` attribute. (:pr:`1185`)
- SDK clients no longer prepend a ``base_path`` to request paths when ``get``, ``put``, ``post``, ``patch``, ``delete``, and ``request`` are called. (:pr:`1185`)
```

Reply:
I was thinking along the lines "SDK client classes no longer define nor prepend a ``base_path`` blah blah blah". Would that be sufficient?

Two sentences seems like more than what's necessary, but that's just how I'm currently thinking...

Reply:
:+1: for "no longer define nor prepend"